src/components/ConnectionStatus.svelte

```diff
@@ -1,33 +1,34 @@
 <script lang="ts">
   import switchIcon from '$lib/assets/switch.png';
   import ConnectProvider from './ConnectProvider.svelte';
-  import { selectedNetwork } from '$lib/stores/networksStore';
+  import { user } from '$lib/stores/userStore';
+  import { isLoggedIn } from '$lib/stores';
 
   $: showConnectProvider = false;
 </script>
 
-<div class="flex items-center gap-6">
-  <!-- {#if connected} -->
-  <!-- TODO: ADD REAL DATA WHEN STORE IS CONNECTED -->
-  <div class="flex items-center gap-2">
-    <div class="w-1 h-1 rounded-full bg-green-success p-1" />
-    <p class="text-md uppercase">{$selectedNetwork?.name}</p>
+{#if $isLoggedIn}
```

Comment:
Looks like we can now delete the TODOs and comments. Also, I think a console log accidentally snuck in from my PR on line 26.

Reply:
fixed